flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/utils/StreamAbstractTestBase.java

```diff
@@ -22,10 +22,14 @@
 import org.apache.flink.test.junit5.MiniClusterExtension;
 
 import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** Base class for unit tests that run multiple tests and want to reuse the same Flink cluster. */
 public class StreamAbstractTestBase {
 
+    protected final Logger log = LoggerFactory.getLogger(getClass());
```

Comment:
nit: The logger is not used anywhere else except for `StreamingWithStateTestBase`. It feels like a premature optimization with the risk that other test classes will miss that there's a protected log field already present and will create their own logger, anyway. That's a proposal you can reject if you want, but to me it would be good enough to have the code change being done in `StreamingWithStateTestBase` as a private field.